### DISCUSSION TITLE
Keep up the work on this awesome project!

### DISCUSSION THREAD

**Discussion by blakeaa827** [jdholtz/auto-southwest-check-in#133] [General]
Just wanted to write an encouraging note to keep up the work on this project.

I just used it to check-in for my wife's flight, and she got 'A' group!

That alone was enough for me to become a sponsor!

Now you just have to stay ahead of Southwest trying to break it... You know they will do their best!

FYI, I have this running in a docker container on an alpine linux server.  Seems to be working great!

**Comment by jdholtz** [comment]
I really appreciate the kind words and your generosity in becoming a sponsor. These things really help me keep improving this software to stay up to date with Southwest. It's unfortunate that Southwest is against this type of software because many people are more inclined to use the airline when there is a project like this script.
